chipy_org/dev_utils/management/commands/makedevdata.py

```diff
@@ -101,6 +103,23 @@ def handle(self, *args, **options):
 
             count += 1
 
+        # Add a bunch of past meetings too for pagination testing
+        for _ in range(10):
+            meeting_id = str(uuid.uuid4())
+            random_days_ago = random.randint(10, 30)
+            meeting_date = now - timedelta(days=random_days_ago)
+            meeting_description = f"Dev meeting{random.randint(1000, 2000)}"
+            meeting_registration_end = meeting_date + timedelta(days=6)
+
```

Comment:
This is slightly odd as we are closing registration after a meeting happens, but I think it's fine for what we're doing here.